diagnostic.md

```diff
@@ -16,13 +16,13 @@ Graces-MacBook-Pro:try grace$
 
 What is the full path to the current directory?
 
-_Replace this text with your answer._
+grace$/wdi/try
```

Comment:
Take a look at line 11. There is some stuff that comes before 'grace'. Also, the $ sign is part of the prompt in bash. It is not a part of the actual directory path.